akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,9 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.actor.default-dispatcher"
```

Comment:
Ah, now I see why you thought plural, because the dispatcher will be for all if there are more than one logger specified. Ok, NVM my comments